src/fontra/views/editor/editor.css

```diff
@@ -334,7 +334,7 @@ body {
   display: flex;
   flex-direction: column;
   gap: 0.4em;
-  overflow: scroll;
```

Comment:
I noticed that this particular group (`.axis-sliders`) is no longer used anywhere. Please delete the whole block now that we're here anyway.